### ISSUE TITLE
ToxicityMetric errors out when actual_output is provided

### ISSUE THREAD

**Issue by pksophos** [confident-ai/deepeval#425]
**Describe the bug**
ToxicityMetric's `measure` method seems to have an incorrect logic in the input validation. If `test_case.actual_output` exists, it errors out, but it should error out only when it's None.

i.e.
```python
        if test_case.input is None or test_case.actual_output:
            raise ValueError("Input or actual output cannot be None")
```
should be:
```python
        if test_case.input is None or test_case.actual_output is None:
            raise ValueError("Input or actual output cannot be None")
```

**To Reproduce**

Run the following code sample (e.g. save as `test_foo.py` and run `deepeval test run test_foo.py`):

```python
import pytest
from deepeval import assert_test
from deepeval.metrics import ToxicityMetric
from deepeval.test_case import LLMTestCase

def test_toxicity():
        metric = ToxicityMetric(threshold=0.5)

        tc = LLMTestCase(
                input="What is 2+2?",
                actual_output="4",
                retrieval_context=[],
        )
        assert_test(tc, [metric])
```

**Expected behavior**
It should not fail.

**Screenshots**
![image](https://github.com/confident-ai/deepeval/assets/119349094/32055e43-ebb5-4189-8e37-2e879ab9abbe)

**Desktop (please complete the following information):**
 - OS: MacOS (but shouldn't matter)
 - Browser: N/A
 - Version [e.g. 22]: deepeval 0.20.50

**Smartphone (please complete the following information):**
N/A

**Additional context**
A simple logic bug in code, but there doesn't seem to be any unit test associated with the code.

**Comment by penguine-ip** [comment]
Thanks for bringing this up! Will change it immediately

**Comment by penguine-ip** [comment]
It's fixed, newest version 0.20.51, run `pip install -U deepeval' for the update, thanks!

**Comment by pksophos** [comment]
That was fast!
